Describe login as a customer from the admin panel (#1324)
magento/merchdocs: src/customers/login-as-customer.md

1. Click <span class="btn">Save Role</span>.

## Login as a customer from the admin panel

1. On the _Admin_ sidebar, go to **Customers** > _All Customers_.

1. Open a user in edit mode.

1. In the **Customer Information** panel, choose the **Account Information** section.

> 1. Set the **Allow remote shopping assistance** toggle to `Yes`.

Reviewer: ```suggestion
1. Set the **Allow remote shopping assistance** to `Yes`.
```

>    After all these steps, the administrator can log in as a user without his permission from the storefront.

Reviewer: ```suggestion
   The administrator can now log in as a user without their permission from the storefront.
```
